id/lessons/advanced/otp-concurrency.md

```diff
@@ -1,11 +1,11 @@
 ---
-version: 0.9.1
+version: 0.10.0
 title: OTP Concurrency
 ---
 
 Kita sudah melihat abstraksi Elixir untuk konkurensi tapi terkadang kita butuh kendali lebih dan untuk itu kita beralih ke perilaku OTP yang mana Elixir dibangun di atasnya.
 
-Dalam pelajaran ini kita akan fokus pada dua bagian penting: Genserver dan GenEvent.
+Dalam pelajaran ini kita akan fokus pada dua bagian penting: Genserver.
```

Comment:
This is the correct translation

Dalam pelajaran ini kita hanya akan fokus pada bagian yang paling penting yaitu Genserver.